.gitignore

```diff
@@ -29,3 +29,433 @@ coverage
 /test-results/
 /playwright-report/
 /playwright/.cache/
+
+# js build files where a ts source file exists
```

Comment:
Ensure we ignore all the temp build files as we don't want to accidentally commit them to git